### ISSUE TITLE
Send reminders for (long) unread messages

### ISSUE THREAD

**Issue by easafe** [typestruck/sprout#1]
Currently emails notifications are sent for messages unread in the last hour. Update notifications to let users know when messages have been unread for 

* right away if the user has been offline for 10+ minutes
* 1 hours
* 1 day
* 1 week
* 1 month

**Comment by Iizuki** [comment]
Did you end up making changes to the emails @easafe ?

**Comment by easafe** [comment]
yes, the mobile app is a better way of notifying users about messages so this issue is no longer relevant. also emails are now sent every 12 hours for the unread messages of the day. recovery, feedback and report emails  use mailjet as well
